frontend/views/pages/Home.vue

```diff
@@ -1,10 +1,15 @@
 <template lang='pug'>
 main.c-splash(data-test='homeLogo')
-  img.logo(src='/assets/images/group-income-icon-transparent.png')
-  h1
-    i18n(data-test='welcomeHome') Welcome to GroupIncome
+  header(v-if='!$store.state.loggedIn' key='title1')
```

Comment:
1. Why does this need a `key='title1'`?
2. If it does need it for some reason, `title1` seems a bit vague?

Reply:
This is a weird bug, but necessary... I'll change the title